docs/wiki/The library - Module design.md

```diff
@@ -273,7 +273,7 @@ param diagnosticEventHubAuthorizationRuleId string = ''
 @description('Optional. Name of the diagnostic event hub within the namespace to which logs are streamed. Without this, an event hub is created for each log category.')
 param diagnosticEventHubName string = ''
 
-@description('Optional. The name of logs that will be streamed.')
+@description('Optional. The name of logs that will be streamed. "allLogs" includes all possible logs for the resource.')
 @allowed([
   <LogsIfAny>
 ])
```

Comment:
Is this the diagnostic template snippet? If so, and based on the changes of this PR, we should probably also update the following lines such as
```suggestion
@description('Optional. The name of logs that will be streamed. "allLogs" includes all possible logs for the resource.')
@allowed([
  'allLogs'
  <LogsIfAny>
])
```

Reply:
Also the default value of `diagnosticLogCategoriesToEnable `should be updated to `['allLogs']`